package-lock.json

```diff
@@ -1,8 +1,10586 @@
 {
   "name": "qpp-measures-data",
   "version": "5.5.7",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I don't think we're ready to upgrade to version 2. Should be able to fix this with running `npm i` with the current npm version for Sub API (`npm use v14.19` I think)

Reply:
Good catch! Fixed